funnel/views/profile.py

```diff
@@ -242,6 +243,61 @@ def view(self) -> ReturnRenderWith:
 
         return ctx
 
+    def jsonify_calendar(jsonData):
+        return jsonify(jsonData['projects'])
+
+    @route('calendar', methods=['GET'], endpoint='calendar')
+    @requestargs('start', 'end')
+    @render_with(
+        {
+            'text/html': 'profile_calendar.html.jinja2',
+            'application/json': jsonify_calendar,
+        }
+    )
+    def org_calendar(
+        self, start: datetime = None, end: datetime = None
```

Comment:
If `None` is a valid value, the types should be `self, start: datetime | None = None, end: datetime | None = None`